{
  try {
    return callable.call();
  }
 catch (  Exception e) {
    throw ProtobufUtil.handleRemoteException(e);
  }
}
